### DISCUSSION TITLE
Tip Button or Donate Button?

### DISCUSSION THREAD

**Discussion by warengonzaga** [web3phl/directory#17] [Brainstorm]
Here's the discussion about whether to add a donation or tip button on the landing page. Feel free to discuss anything about this here below. What do you think? 🤔

**Comment by davePawww** [comment]
Not really sure what's the difference but it's a good addition. Maybe something like "Buy us coffee" 😆

**Reply by warengonzaga** [reply]
Make sense to me 😅

**Comment by chefallan** [comment]
okay here is the sketch from the paint editor. parang ganito lang
![Untitled](https://user-images.githubusercontent.com/31358319/180091088-b5dfa9e9-e5a7-4be8-b180-451557e6dc4b.png)

under ng heart/like button, share and tip/donate. ay comment section na.

**Reply by warengonzaga** [reply]
I guess this is a good idea then combine the idea with @kquirapas which is located in the main navbar too. What do you think?

**Reply by chefallan** [reply]
> I guess this is a good idea then combine the idea with @kquirapas which is located in the main navbar too. What do you think?

my bad for this, not really paying attention to what the discussion is all about. I was thinking about the web3 directory will be a tutorial/teachings about web3 per article that is published to the directory will have specific pages that have the characteristics that I draw above. 

but all of you mean about how can others support the development of the project web3ph directory. So yeah it is a good suggestion that we put the donate button along with the navbar.

**Reply by warengonzaga** [reply]
> my bad for this, not really paying attention to what the discussion is all about. I was thinking about the web3 directory will be a tutorial/teachings about web3 per article that is published to the directory will have specific pages that have the characteristics that I draw above.

Oh, I didn't notice it too 😅

For the design docs, you can find it on our opensource channel in the Discord server, pinned it there.
To give you context, the directory is a digital directory list of awesome Web3 projects created and led by Filipinos. Here's the example for your reference: https://simpleicons.org/

**Reply by chefallan** [reply]
> > my bad for this, not really paying attention to what the discussion is all about. I was thinking about the web3 directory will be a tutorial/teachings about web3 per article that is published to the directory will have specific pages that have the characteristics that I draw above.
> 
> Oh, I didn't notice it too 😅
> 
> For the design docs, you can find it on our opensource channel in the Discord server, pinned it there. To give you context, the directory is a digital directory list of awesome Web3 projects created and led by Filipinos. Here's the example for your reference: https://simpleicons.org/

the suggestion of ivan is here: [Add a Donate button to each project, so anyone who browses and finds a project interesting enough can provide a small financial donation. But this can be optional also. Coordinate with the project owner if they want to allow this.]

so maybe we will have a different discussion for this suggestion of ivan and this discussion that we have right now.

all in all i agree with @kquirapas idea to add the button along with the navbar.

**Comment by kquirapas** [comment]
sa may navbar if ever para kita agad 🤝

**Reply by warengonzaga** [reply]
Let's combine this with the idea with @itsnalla? 🤔

**Reply by jnebab** [reply]
I am not sure if I am following, why navbar? how do we recognize the specific project we want to tip or donate to?

**Reply by jnebab** [reply]
![image](https://user-images.githubusercontent.com/6265768/180155347-1fc9fb62-929f-4cad-8d69-97588da56ca4.png)

I would assume the idea is like this?

**Reply by kquirapas** [reply]
Hey Ivan, great question.

As seen on the discussion post above we're adding the button in the landing page, not per project

Hence the donations/tips go to the directory team

Hope this clarifies things.

**Reply by kquirapas** [reply]
So like "Donate for the team who keepd this directory alive" ganern 👌🏾

**Reply by jnebab** [reply]
I see, when I wrote this to the suggestions section is to donate to a specific project listed in the directory.

**Reply by kquirapas** [reply]
Yeah that's why it was confusing at first haha

But the donate idea is good 🤝

**Reply by jnebab** [reply]
ok ok. goods

**Reply by warengonzaga** [reply]
I like the idea of donation per project! @jnebab @kquirapas let's add it to the feature request? Because not all of the listed items need donations or tips. Maybe let's add the donate button for the community instead.

**Reply by jnebab** [reply]
> I like the idea of donation per project! @jnebab @kquirapas let's add it to the feature request? Because not all of the listed items need donations or tips. Maybe let's add the donate button for the community instead.

I was thinking on suggesting this with web3/thirdweb integration if its possible. Like we show a Donate {amount in MATIC} button if the owner of the listed directory provided their wallet address. Then before a reader/viewer/user can donate, they need to connect their wallet, then after, we kinda automate the sending of MATIC to the wallet address provided by the owner of that certain listed project

**Reply by warengonzaga** [reply]
> > I like the idea of donation per project! @jnebab @kquirapas let's add it to the feature request? Because not all of the listed items need donations or tips. Maybe let's add the donate button for the community instead.
> 
> I was thinking on suggesting this with web3/thirdweb integration if its possible. Like we show a Donate {amount in MATIC} button if the owner of the listed directory provided their wallet address. Then before a reader/viewer/user can donate, they need to connect their wallet, then after, we kinda automate the sending of MATIC to the wallet address provided by the owner of that certain listed project

I think, how about once they donate they will receive a corresponding NFT of that particular listed item. I'm thinking more like signature-based minting.

**Reply by kquirapas** [reply]
Future features are easy to attach. We can add them as feat reqs then focus on getting the base app first 👍

The directory is an information source first, additional features second.

what matters now the most is they have a reliable information source 👌🏾

We can easily extend and improve as we go

What do you guys think?

**Reply by warengonzaga** [reply]
> Future features are easy to attach. We can add them as feat reqs then focus on getting the base app first 👍
> 
> The directory is an information source first, additional features second.
> 
> what matters now the most is they have a reliable information source 👌🏾
> 
> We can easily extend and improve as we go
> 
> What do you guys think?

Yeah, we can do that, I'm finalizing things so I can create some feature requests in one shot.
